backend/src/api/Controllers/BlogController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using api.Models;
```

Comment:
Why are We creating this? It will need to be deleted as soon as We start working on the actual app code. Do You think it is a good idea? Maybe let's just add `/login` route and a `Users` table with Id, email, password, and token?

Reply:
I mean to combine this with issue #18